test_scripts/UpdateVideoStreamingCapabilities/002_2_OnSystemCapabilityUpdated_without_AddVSC.lua

```diff
@@ -0,0 +1,29 @@
+-- https://adc.luxoft.com/jira/browse/FORDTCN-6978
+
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local common = require('test_scripts/UpdateVideoStreamingCapabilities/common')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local expected = 1
+local isSubscribe = true
+
+local vsc = common.cloneTable(common.anotherVideoStreamingCapabilityWithOutAddVSC)
+vsc.additionalVideoStreamingCapabilities = nil
```

Comment:
`anotherVideoStreamingCapabilityWithOutAddVSC` parameter is defined as value wihout additional VSC

Reply:
@aderiabin Yes, you are right. But with this line of code we can be sure that it is.